packages/blade/src/components/Input/BaseInput/BaseInputTagSlot.native.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { ScrollView, Text as RNText } from 'react-native';
+import styled from 'styled-components';
+import type { BaseInputTagSlotProps } from './types';
+import { BASEINPUT_DEFAULT_HEIGHT, BASEINPUT_WRAPPER_MAX_HEIGHT } from './baseInputConfig';
+import BaseBox from '~components/Box/BaseBox';
+import { makeSize } from '~utils';
+import { size } from '~tokens/global';
+
+const StyledScrollView = styled(ScrollView)((_props) => {
+  return {
+    display: 'flex',
+    flexDirection: 'column',
+    // gap is still not working in RN for some reason
```

Comment:
What why?

Reply:
Maybe it had something to do with ScrollView. Anyway removed it. Can figure out later